lib/stat_board/graph_helper.rb

```diff
@@ -14,11 +14,27 @@ def first_day_ever
     # along the date_range, on every displayed interval
     def resources_by_date(klass_name)
       klass       = klass_name.to_s.constantize
-      created_ats = klass.pluck(:created_at).compact.map(&:to_date)
-
-      date_range.step(date_steps).map do |date|
-        created_ats.count{|c_a| c_a < date}
-      end.to_s
+      created_ats = klass.order(:created_at).pluck(:created_at).compact
+      steps       = date_range.step(date_steps).map(&:end_of_day)
+      counts      = created_ats.reduce(Hash.new(0)) do |counts, timestamp|
+        cutoff = steps[counts[:index]]
```

Comment:
maybe rename `cutoff` to `end_of_day` ? It's not really a "cutoff" until the very end when one isn't present.